Assets/_BForBoss/_Core/Scripts/UserInterface/UseCases/QuitGameUseCase.cs

```diff
@@ -0,0 +1,14 @@
+namespace BForBoss
+{
+    public class QuitGameUseCase
```

Comment:
Why call them UseCase and not behaviours? i.e. `QuitGameBehaviour.cs`.

Reply:
This is a specific UI term for frontend development. Basically UseCases can be reusableto any other UI places.
E.g. when there's a main menu with a quit button, we just need to create an object of this class and use it.  

https://betterprogramming.pub/the-clean-architecture-beginners-guide-e4b7058c1165